cex-collection/js-example/index.js

```diff
@@ -3,8 +3,11 @@ import {ethers} from 'ethers'
 import {PaymasterClient, SponsorClient, WhitelistType} from 'megafuel-js-sdk'
 
 async function cexDoGaslessTransferTx() {
+  const chainID = process.env.CHAIN_ID
```

Comment:
can we put chainid into `sponsorUrl` so that we dont have to concat here?

Reply:
updated the readme 
![image](https://github.com/user-attachments/assets/3fa0b366-a4d3-4e46-9046-eeef7de83071)
`const paymasterClient = PaymasterClient.newPrivatePaymaster(sponsorUrl, policyUUID,  ethers.Network.from(Number(chainID)), {staticNetwork: ethers.Network.from(Number(chainID))})
 `
and directly use sponsorUrl to init client